EMacco/abarian-backend
# #170919015 Reset password

#### What does this PR do?
This PR allows a user to be able to change his/her password

#### Description of Task to be completed?
- validate token to make sure it belongs to user
- validate password
- confirm token validity
- send notification email

#### How should this be manually tested?
- checkout to this branch
- make a reset password request using `POST /auth/forgot_password`
- open email and copy the token
- make a post request to `/auth/reset_password&token=enter_the_token&email=enter_your_email`
- your password should be reset

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170919015](https://www.pivotaltracker.com/story/show/170919015)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
